content/post/community-india-is-back.md

```diff
@@ -34,14 +34,14 @@ Please go through the [Code of Conduct](https://communityday.awsug.in/code-of-co
 
 AWS Community Day 2020 workshops last for 4 hours and are listed below:
 
-* **Containers:** The workshop will be lead by Sandip Das ( AWS Container Hero) and Dipali Kulshrestha (AWS Community Leader) on 17th October at 2:00 pm IST. The workshop will be for 4 hours.
-* **Machine Learning:** The workshop will be lead by Jayesh Ahire (AWS ML Hero) and Sukanya Mandal (AWS reInvent 2019 Diversity Scholarship Winner) on 17th October at 2:00 pm IST. The workshop will be for 4 hours.
-* **Serverless:** The workshop will be lead by Bhuvaneswari Subramani and Dhaval Nagar on 17th October at 2:00 pm IST. The workshop will be for 4 hours.
-* **Alexa:** The workshop will be lead by Karthik Ragubathy(Solutions Architect) with the Alexa team on 17th October at 2:00 pm IST. The workshop will be of 4 hours
+* **Containers:** The workshop will be lead by Sandip Das ( AWS Container Hero) and Dipali Kulshrestha (AWS Community Leader) on 17th October at 2:00 pm IST. 
+* **Machine Learning:** The workshop will be lead by Jayesh Ahire (AWS ML Hero) and Sukanya Mandal (AWS reInvent 2019 Diversity Scholarship Winner) on 17th October at 2:00 pm IST. 
+* **Serverless:** The workshop will be lead by Bhuvaneswari Subramani and Dhaval Nagar on 17th October at 2:00 pm IST. 
+* **Alexa:** The workshop will be lead by Karthik Ragubathy(Solutions Architect) with the Alexa team on 17th October at 2:00 pm IST. 
 
   ## **KEYNOTE SPEAKER**
 
-  .![](/images/jeffbarr.jpg "eff Barr, Chief Evangelist, AWS")
+  ## .![](/images/jeffbarr.jpg "eff Barr, Chief Evangelist, AWS")
```

Comment:
"eff Barr, Chief Evangelist, AWS" => Jeff Barr

Reply:
Corrected!

Reply:
@dA505819 still need one more fix
![image](https://user-images.githubusercontent.com/25424/93582761-98698f00-f9ab-11ea-938d-1466bdf7e48c.png)

Reply:
I fixed this